src/Hl7.Fhir.Shims.STU3AndUp/Serialization/FhirJsonParser.cs

```diff
@@ -16,52 +16,21 @@
 
 namespace Hl7.Fhir.Serialization;
 
-public class FhirJsonParser(ParserSettings? settings = null) : BaseFhirParser
+public class FhirJsonParser(ParserSettings? settings = null)
+    : BaseFhirJsonParser(ModelInfo.ModelInspector, settings)
 {
-    public ParserSettings Settings { get; set; } = settings ?? new ParserSettings();
+    /// <inheritdoc cref="FhirXmlParser.DEFAULT" />
+    public static readonly FhirJsonParser DEFAULT = new();
 
-    /// <inheritdoc cref="ParseAsync{T}(string)" />
-    public T Parse<T>(string json) where T : Base => (T)Parse(json, typeof(T));
+    /// <inheritdoc cref="FhirXmlParser.STRICT" />
+    public static readonly FhirJsonParser STRICT = new(new ParserSettings().UsingMode(DeserializationMode.Strict));
 
-    public async Tasks.Task<T> ParseAsync<T>(string json) where T : Base
-        => (T)await ParseAsync(json, typeof(T)).ConfigureAwait(false);
+    /// <inheritdoc cref="FhirXmlParser.RECOVERABLE" />
+    public static readonly FhirJsonParser RECOVERABLE = new(new ParserSettings().UsingMode(DeserializationMode.Recoverable));
 
-    /// <inheritdoc cref="ParseAsync{T}(JsonReader)" />
-    public T Parse<T>(JsonReader reader) where T : Base => (T)Parse(reader, typeof(T));
+    /// <inheritdoc cref="FhirXmlParser.BACKWARDSCOMPATIBLE" />
+    public static readonly FhirJsonParser BACKWARDSCOMPATIBLE = new(new ParserSettings().UsingMode(DeserializationMode.BackwardsCompatible));
 
-    public async Tasks.Task<T> ParseAsync<T>(JsonReader reader) where T : Base
-        => (T)await ParseAsync(reader, typeof(T)).ConfigureAwait(false);
-
-    /// <inheritdoc cref="ParseAsync(string, Type)" />
```

Comment:
This stuff has been moved to (legacy) extension methods you have already seen above.